source/includes/plugin_settings/_02-show.md.erb

```diff
@@ -51,12 +51,13 @@ ETag: "05548388f7ef5042cd39f7fe42e85735"
 }
 ```
 
-Gets the stored plugin settings for the specified plugin id.
+Gets the stored plugin settings for the specified plugin id. Be aware that a settings object is created on demand and may not be availble!
```

Comment:
Would it be OK to say "Gets the stored plugin settings for the specified plugin id, if the plugin is configured.", since that is what the last line here says?

Reply:
yes, describes the behavior in my opinion!